python/lsst/ctrl/ingestd/rseButler.py

```diff
@@ -40,7 +40,7 @@ def __init__(self, repo: str):
 
         self.butler = Butler(repo, writeable=True)
 
-    def create_entry(self, butler_file: str, sidecar: dict) -> FileDataset:
```

Comment:
In the parameter description below (line 62) `sidecar`'s type is still `dict`. Please fix.